summaries/summary-lec.toml

```diff
@@ -0,0 +1,1595 @@
+# Summary for Libra subset 'lec'. @generated by x.
+# To regenerate, run 'cargo x generate-summaries'.
```

Comment:
Actually thinking about this more, we probably can't merge this yet until we can block landing  if these summary files are out of date. The last thing we want is the action reporting differences when a PR doesn't actually include any (but a previously merged PR into master had differences but didn't update these files)

Reply:
So we probably just need to add a step in CI that just runs `cargo x generate-summaries` and ensures that no files are modified after that.

Reply:
I'll add something to fail the PR if the sumamries don't match.